iroha/iroha.py

```diff
@@ -254,6 +256,10 @@ def query(self, name, counter=1, creator_account=None,
         :param created_time: query creation timestamp in milliseconds
         :param page_size: a non-zero positive number, size of result rowset for queries with pagination
         :param first_tx_hash: optional hash of a transaction that will be the beginning of the next page
+        :param first_tx_time: optional time of first transaction (will not be included)
+        :param last_tx_time: optional time of last transaction (will not be included)
+        :param first_tx_height: optional block height of first transaction (will not be included)
+        :param last_tx_height: optional block height of last transaction (will not be included)
```

Comment:
I suggest to remove text `(will not be included)`, because `first_tx_hash` is also optional, but doesn't contain the text